.konflux/ubi.repo

```diff
@@ -0,0 +1,62 @@
+[ubi-9-for-x86_64-baseos-rpms]
+name = Red Hat Universal Base Image 9 (RPMs) - BaseOS
+baseurl = https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi9/9/$basearch/baseos/os
+enabled = 1
+gpgkey = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+gpgcheck = 1
```

Comment:
Do we need to maintain this file at all if all we need is to fetch packages from BaseOS and Appstream? Recently, we did the following to go about this just fine: https://github.com/hermetoproject/integration-tests/blob/multiple/gomod-and-npm/rpms.in.yaml#L3-L6

Reply:
We need a small [tweak](https://github.com/brunoapimentel/hermeto/commit/f3114611956d5344a6d1e0217f0a90ba00b3d1a2) to have the repoids that will end up in the SBOM to comply with Konflux's rules. So we can either keep this file as is, or have a `sed` command to update the resulting `rpms.lock.yaml` file.

Having the file here is probably a little simpler, but I have no objections about going the other way.